projects/step-core/src/lib/components/predefined-options-input/predefined-options-input.component.scss

```diff
@@ -5,6 +5,10 @@ button.mat-menu-item {
   line-height: 1;
 }
 
+input.hide-disabled:disabled {
+  background: white;
+}
```

Comment:
@neogucky  In case if you will apply the readonly attribute, this can be removed

Reply:
Thanks, makes sense!